### PR TITLE
Checkout

### PR DESCRIPTION
**Pull request by yieknee**

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -1,5 +1,20 @@
 class Order < ApplicationRecord
   has_many :order_items
+  validates :name, :email, :mailing_address, :cc_name, :cc_cvv, :cc_exp, :cc_number, :zipcode, :status, presence: true
+  validates :email, format: { with: /\w+@\w+\.\w+/, message: "Please enter valid email address" }
+  validates :cc_cvv, :cc_number, :zipcode, numericality: { only_integer: true }
+  validates :cc_number, length: {is: 15}
+  validates :cc_cvv, length: {is: 3}
+  validates :zipcode, length: {is: 5}
```
Comment (cojenco) [inline app/models/order.rb:9]: cc_number length validation should be 16? Once we merge, let's check if the tests and orders#show need to be revised.

File: app/models/order.rb [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
 
+  def order_total
+    total = 0
+    self.order_items.each do |order_item|
+      price = order_item.product.price
+      total += price
+    end
+
+    return (total * 1.1)
+  end
```
Comment (cojenco) [inline app/models/order.rb]: It seems like "quantity" is missing in this calculation. Please note that I made a #subtotal instance method in the OrderItem model, which we should be able to incorporate into this method. We will need to merge both PR's first👍

**Review by cojenco — APPROVED** [review]
LGTM!!!🏆 Once PR#11 and PR#12 are merged, we can incorporate order_items#subtotal instance method into the order_total method here. And tests.

**Review by aecombs — CHANGES_REQUESTED** [review]
Code looks good -- echoing Cathy for cc_number validations and order_total method, and also add tests for controller actions and model methods/validations/relations

File: app/models/order.rb (hunk 2 above)
Comment (yieknee) [reply app/models/order.rb]: i've made the change

**Pull request merged**